code/modules/cargo/exports/materials.dm

```diff
@@ -29,65 +29,57 @@
 
 // Materials. Nothing but plasma is really worth selling. Better leave it all to RnD and sell some plasma instead.
 
-// Bananium. Exporting it makes the clown cry. Priceless.
 /datum/export/material/bananium
-	cost = 5000
+	cost = 1000
 	material_id = MAT_BANANIUM
 	message = "cm3 of bananium"
 
-// Diamonds. Rare and expensive.
 /datum/export/material/diamond
-	cost = 2500
+	cost = 500
 	material_id = MAT_DIAMOND
 	message = "cm3 of diamonds"
 
-// Plasma. The oil of 26 century. The reason why you are here.
 /datum/export/material/plasma
-	cost = 300
+	cost = 200
 	k_elasticity = 0
 	material_id = MAT_PLASMA
 	message = "cm3 of plasma"
 
-// Uranium. Still useful for both power generation and nuclear annihilation.
 /datum/export/material/uranium
-	cost = 400
+	cost = 100
 	material_id = MAT_URANIUM
 	message = "cm3 of uranium"
```

Comment:
Why the massive price reduction for uranium? It's reasonably rare and the justification in the comment seems reasonable.